### PR TITLE
[OCPCLOUD-1732] Add test-e2e-periodic target in the makefile

### PR DESCRIPTION
**Pull request by Fedosin**
This PR adds a target to run e2e periodic tests only.

### CODE REVIEW

**Comment by JoelSpeed** [comment]
For each test that you've marked as a periodic, can you please explain why you think this warrants being a periodic rather than running on presubmit?

I would like to be very deliberate about the tests that we mark as periodic, they should have good reasons behind them.

For example, it may be useful to make a test plan similar to the one for the CPMS which explains why each of these tests should be periodic and the behavioural subsets we cover in presubmits

**Comment by lobziik** [comment]
Is this still actual after https://github.com/openshift/cluster-api-actuator-pkg/pull/258 was merged?

There I marked proxy test as periodic: https://github.com/openshift/cluster-api-actuator-pkg/pull/258/files#diff-ddf400001089ed146e8291e2e81b9fa305e9e371d81a389561ee6c61a11b8a5bR229

/hold

File: Makefile [hunk 1]
```diff
@@ -74,6 +73,10 @@ build-e2e:
 test-e2e: ## Run openshift specific e2e test
 	hack/ci-integration.sh $(GINKGO_ARGS) --label-filter='!(periodic || spot-instances)' -p
 
+.PHONY: test-e2e-periodic
+test-e2e-periodic: ## Run openshift specific periodic e2e test
+	hack/ci-integration.sh $(GINKGO_ARGS) --label-filter='periodic && !spot-instances' -p
```
Comment (lobziik) [inline Makefile:78]: think `--label-filter='periodic'` should be enough

**Review by lobziik — CHANGES_REQUESTED** [review]
Think we don't need to explicitly exclude spot-tests

**Comment by JoelSpeed** [comment]
/override ci/prow/e2e-azure-operator

The autoscaler test that's failing is unrelated to this change. I can see the proxy test is still being skipped and the new make target looks good to me, don't think we need this to pass

**Pull request merged**